packages/shared/src/components/profile/UpvoterListPlaceholder.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactElement, memo } from 'react';
+import { PlaceholderList } from '../cards/PlaceholderList';
+
+export interface UpvoterListPlaceholderProps {
+  placeholderAmount?: number;
+}
+
+export function UpvoterListPlaceholder({
+  placeholderAmount = 5,
```

Comment:
Instead of using a hardcoded default here, we can use the number of upvotes the article/comment received as a guess for how many upvotes we're going to show. One caveat is that we need to limit this number to the number of rows per maximum height modal. For example, if we have 100 upvoters we don't need 100 placeholder rows because there's not enough room for them anyway on the screen. So please check what's the maximum number of items when the modal is maximized.

Reply:
It is 5, so I'll just use that value as the ceiling for guessing how many placeholders to show.